### DISCUSSION TITLE
Value of one field depends on calculating two fields from useFieldArray

### DISCUSSION THREAD

**Discussion by nenaddimitrijevic89** [react-hook-form/react-hook-form#9022] [Q&A]
Is there possibility to get value for one field in react hook form (this input is read only) through calculation two fields from useFieldArray hook? Specifically I have quantity and price field and want to calculate total field, how to achieve that? Thanks
![hook-form](https://user-images.githubusercontent.com/60787476/189553280-f44e57f3-2c31-481a-b855-115777f5d0dd.PNG)

**Comment by bluebill1049** [comment]
could you use `useWatch` hook to subscribe to the form values which you want to?

**Reply by nenaddimitrijevic89** [reply]
yes, I am using `useWatch`. But I am adding fields dinamically, so every field has its own `total` which should be calculated based on its `price` and `quantity`. Cant figure out in which way to implement this :( @bluebill1049

**Reply by nenaddimitrijevic89** [reply]
@bluebill1049 any help? :(

**Reply by bluebill1049** [reply]
pretty sure when you append you get the latest value from useWatch: https://codesandbox.io/s/usefieldarray-conditional-forked-wl5czz?file=/src/App.tsx
